### ISSUE TITLE
Update FontAwesome to 4.6.1

### ISSUE THREAD

**Issue by zhedar** [TheCoder4eu/BootsFaces-OSP#353]
As [FA 4.6.1 is available now](https://github.com/FortAwesome/Font-Awesome/releases), maybe we should update to that version with 0.8.2?

**Comment by stephanrauh** [comment]
Doesn't sound risky to me, so you've got my "go!". Do you know what's changed since 4.6.0?

**Comment by zhedar** [comment]
We're currently running 4.5. Apparently 4.6 [added 23 new icons](http://fontawesome.io/whats-new/), but something was wrong with 4.6, so 4.6.1 followed in an instant.

**Comment by zhedar** [comment]
I just found out how to upgrade the FA version we use and apparently, we only get it from CDN and don't bundle it locally. It wasn't in a property file, where I looked for it. I will include this small change in my current PR.
If we set the load from CDN context param to `false` and FA isn't included by another library (e.g. PrimeFaces) or by the user himself we won't load it at all, is that working as intended?
